packages/did/src/DidDetails/FullDidDetails.spec.ts

```diff
@@ -234,17 +234,17 @@ describe('When creating an instance from the chain', () => {
         const extrinsic = augmentedApi.tx.utility.batch([
           await augmentedApi.tx.ctype.add('test-ctype'),
         ])
-        await expect(async () =>
-          Did.authorizeBatch({
-            did: fullDid,
-            batchFunction: augmentedApi.tx.utility.batchAll,
-            extrinsics: [extrinsic, extrinsic],
-            sign,
-            submitter: keypair.address,
-          })
-        ).rejects.toMatchInlineSnapshot(
-          '[DidBuilderError: Can only batch extrinsics that require a DID signature]'
-        )
+        const batchFunction =
+          jest.fn() as unknown as typeof mockedApi.tx.utility.batchAll
```

Comment:
I thought mockedApi.tx.utility.batchAll is already a jest.fn()

Reply:
Could we cast it to this type in mockedApis then?

Reply:
`jest.fn()` needs to have the type of `mockedApi.tx.utility.batchAll` so that `jest.fn()` can be used as a batchFunction. I'm not sure if we can change anything here.

Reply:
Oh, now I see this can’t be done there